app/src/main/java/com/togetherjava/tjplays/listeners/commands/Game2048Command.java

```diff
@@ -44,7 +45,10 @@ public void onSlashCommandInteraction(SlashCommandInteractionEvent event) {
     public void onButtonInteraction(ButtonInteractionEvent event) {
         String buttonId = event.getButton().getId();
         if (!buttonId.startsWith(COMMAND_NAME)) return;
-
+        if (!event.getUser().getId().equals(initialUser)) {
+            event.reply(event.getUser().getAsMention()+" you can't interact with this game.").queue(); 
```

Comment:
reply with ephemeral

Reply:
Okay I need to research this in JDA

Reply:
Okay I see I just need to add the setEphemeral method after

Reply:
> Okay I see I just need to add the setEphemeral method after

yes

Reply:
and with sending the user id in button id u can do `buttonId.contains(event.getUser().getId())`

Reply:
Fixed!